src/components/home/Divider.jsx

```diff
@@ -1,5 +1,24 @@
+import Image from "next/image";
+import profile from "@/public/prishi-logo.webp";
+
 const Divider = () => {
-  return <div>Divider</div>;
+  return (
+    <div className="flex items-center h-auto w-full">
+      <div className="flex flex-col items-end w-3/5 ">
```

Comment:
avoid using arbitrary margins to space related elements. you can add spacing between elements in a flex container using gap. add gap-4 here and remove mt-4 below